mrp_production_serial_matrix/wizards/mrp_production_serial_matrix_line.py

```diff
@@ -14,14 +14,16 @@ class MrpProductionSerialMatrix(models.TransientModel):
     production_id = fields.Many2one(related="wizard_id.production_id")
     component_id = fields.Many2one(comodel_name="product.product")
     component_column_name = fields.Char()
-    finished_lot_id = fields.Many2one(comodel_name="stock.production.lot")
+    finished_lot_id = fields.Many2one(comodel_name="stock.lot")
     finished_lot_name = fields.Char()
     component_lot_id = fields.Many2one(
-        comodel_name="stock.production.lot",
+        comodel_name="stock.lot",
+        # TODO: The domain seems to not be affecting.
+        # https://github.com/OCA/manufacture/pull/1443#issuecomment-2582585167
```

Comment:
this has been fixed, you can remove the comment, right?

Reply:
True, done!